lib/fastlane/plugin/ddg_apple_automation/helper/ddg_apple_automation_helper.rb

```diff
@@ -8,6 +9,16 @@ module Helper
     class DdgAppleAutomationHelper
       ASANA_API_URL = "https://app.asana.com/api/1.0"
       ERROR_ASANA_ACCESS_TOKEN_NOT_SET = "ASANA_ACCESS_TOKEN is not set"
+
+      def self.load_asset_file(file)
```

Comment:
Just a note that this isn't strictly "loading" the file, but only constructing file path. We could have loading here, in which case we need `File.read` at the end, or we can keep it as is and rename to `path_for_asset_file` for instance (?)